components/count-badge/demo/count-badge.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../demo/demo-page.js';
+			import '../count-badge.js';
+        </script>
+	</head>
+	<body unresolved>
+		<d2l-demo-page page-title="d2l-count-badge">
+
+			<h2>Small Notification Badge</h2>
+			<d2l-demo-snippet>
+				<template>
+					<d2l-count-badge badge-size="small" badge-type="notification" number="1"></d2l-count-badge>
+					<d2l-count-badge badge-size="small" badge-type="notification" number="10"></d2l-count-badge>
+					<d2l-count-badge badge-size="small" badge-type="notification" number="100"></d2l-count-badge>
```

Comment:
This may be coming in one of the later stories, but what's the accessibility plan in terms of providing both a better label for the count (e.g. "100 ungraded activities") and also optionally linking the count with other elements (maybe via `aria-describedby` or something)?

Reply:
I haven't put much thought into the accessibility plan, it will be in a later story for sure 😄 Stacey mentioned that the filter does have accessible text within the menu-item, so it's not needed urgently for filter.